scripts/bot/run-bot.js

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env node
+const doc = `
+Usage:
+    ./run-bot.js [options]
+
+Options:
+    -h --host=<host>  Hubs host [default: localhost:8080]
+    -r --room=<room>  Room id [default: 234234].
+    -h --help         Show this screen.
+`;
+
+const docopt = require("docopt").docopt;
+const options = docopt(doc);
+
+const puppeteer = require("puppeteer");
+const querystring = require("query-string");
```

Comment:
not sure which version of Node is required for development and production (though the `.travis.yml` file specifies Node version 9). in Node version 8+, you can use `const { URL } = require("url");`